docs/config.md

```diff
@@ -152,8 +152,27 @@ spec:
     subProcessMaxHeapSize: 200
 ```
 
+If the sidecar's resource requests are not specified, they are set with the following defaults:
+
+| Request | Quantity |
+|---------|----------|
+| defaultReportCpuRequest | 128m |
```

Comment:
I think it's probably better to specify this in plain English instead of using the constant names. Something like "Reports Container CPU/Memory".

Reply:
Right! I will change that!